packages/flutter/test/rendering/proxy_getters_and_setters_test.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('RenderConstrainedBox getters and setters', () {
+    RenderConstrainedBox box = new RenderConstrainedBox(additionalConstraints: new BoxConstraints.tightFor(height: 10.0));
+    expect(box.additionalConstraints, new BoxConstraints(minHeight: 10.0, maxHeight: 10.0));
+    box.additionalConstraints = new BoxConstraints.tightFor(width: 10.0);
+    expect(box.additionalConstraints, new BoxConstraints(minWidth: 10.0, maxWidth: 10.0));
+  });
+
+  test('RenderLimitedBox getters and setters', () {
+    RenderLimitedBox box = new RenderLimitedBox();
+    expect(box.maxWidth, double.INFINITY);
+    expect(box.maxHeight, double.INFINITY);
+    box.maxWidth = 0.0;
+    box.maxHeight = 1.0;
+    expect(box.maxHeight, 1.0);
+    expect(box.maxWidth, 0.0);
+  });
+
+  test('RenderAspectRatio getters and setters', () {
+    RenderAspectRatio box = new RenderAspectRatio(aspectRatio: 1.0);
+    expect(box.aspectRatio, 1.0);
+    box.aspectRatio = 0.2;
+    expect(box.aspectRatio, 0.2);
+    box.aspectRatio = 1.2;
```

Comment:
It's also not an error we've made often (at all?).  I'm ok skipping checking it.